RELEASE.md

```diff
@@ -0,0 +1,28 @@
+# Release Process
+
+This document describes the process for releasing a new version of the `actors-utils` project.
+
+## Current State
+
+1. On a release pull request creation, a [Release Checker](.github/workflows/release-check.yml) workflow will run. It will perform the following actions:
```

Comment:
```suggestion
1. Create a pull request which updates the [`version` in one or more `Cargo.toml` files](https://github.com/search?q=repo%3Afilecoin-project%2Factors-utils+path%3ACargo.toml+%2F%5Eversion+%3D%2F&type=code).
   - Title the PR `build: release Ɑ for Ƅ` where:
      - `Ɑ` is what is being released like "all crates"
      - `Ƅ` is the cause for the release like "fvm vX.Y.Z" or "nvXX"
2. On a release pull request creation, a [Release Checker](.github/workflows/release-check.yml) workflow will run. It will perform the following actions:
```

I wanted to make clear what files are to be edited and to give guidance on how to name the PR.